src/js/tabs/options.js

```diff
@@ -105,6 +106,18 @@ options.initCliAutoComplete = function () {
         }).change();
 };
 
+options.initAutoConnectConnectionTimeout = function () {
+    ConfigStorage.get('connectionTimeout', function (result) {
```

Comment:
Reading this from local storage in 3 different places makes this quite hard to maintain - this should probably be centralised.

Reply:
Yes this is lazy and reusing the same value on different occasions. But it works for some users as it's an option you would have to specify.

Reply:
Could add a `getConnectionTimeOut` function to centralize.

Reply:
@mikeller Do you want me to make two separate values so this could be changed individual for both port_handler `auto-connect` and serial_backend `reinitializeConnection` and where should I centralize (put) the function `bool getconnectionTimeout(func)` to read the respective value?